### PR TITLE
Update fallbackListData.ts

### PR DESCRIPTION
**Pull request by AhmedSabry957**

### CODE REVIEW

**Comment by MattIPv4** [comment]
As I mentioned *many* times in Discord, instead of just adding your list, this needs to be updated to match the latest BotBlock data again from https://botblock.org/api/lists?filter=true:

```js
export default {
  'arcane-center.xyz': {
    api_docs: 'https://arcane-center.xyz/documentation',
    api_post: 'https://arcane-center.xyz/api/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: 'shard_count',
    api_shards: null,
    api_get: null,
    api_all: null,
  },
  'bladebotlist.xyz': {
    api_docs: null,
    api_post: 'https://bladebotlist.xyz/api/bots/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: 'shard_count',
    api_shards: null,
    api_get: 'https://bladebotlist.xyz/api/bots/:id',
    api_all: null,
  },
  'blist.xyz': {
    api_docs: 'https://blist.xyz/docs/',
    api_post: 'https://blist.xyz/api/v2/bot/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: 'shard_count',
    api_shards: null,
    api_get: 'https://blist.xyz/api/v2/bot/:id',
    api_all: null,
  },
  'botlist.space': {
    api_docs: 'https://docs.botlist.space/',
    api_post: 'https://api.botlist.space/v1/bots/:id',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: 'shards',
    api_get: 'https://api.botlist.space/v1/bots/:id',
    api_all: 'https://api.botlist.space/v1/bots',
  },
  'botlists.com': {
    api_docs: 'https://botlists.com/api/docs',
    api_post: 'https://botlists.com/api/bot',
    api_field: 'guild_count',
    api_shard_id: null,
    api_shard_count: 'shard_count',
    api_shards: null,
    api_get: 'https://botlists.com/api/bot',
    api_all: null,
  },
  'bots.discordlabs.org': {
    api_docs: 'https://docs.discordlabs.org/',
    api_post: 'https://bots.discordlabs.org/v2/bot/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: 'shard_count',
    api_shards: null,
    api_get: 'https://bots.discordlabs.org/v2/bot/:id',
    api_all: null,
  },
  'bots.idledev.org': {
    api_docs: null,
    api_post: 'https://bots.idledev.org/api/bot/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://bots.idledev.org/api/bot/:id/info',
    api_all: null,
  },
  'bots.ondiscord.xyz': {
    api_docs: 'https://bots.ondiscord.xyz/info/api',
    api_post: 'https://bots.ondiscord.xyz/bot-api/bots/:id/guilds',
    api_field: 'guildCount',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: null,
    api_all: null,
  },
  'botsdatabase.com': {
    api_docs: 'https://docs.botsdatabase.com/',
    api_post: 'https://api.botsdatabase.com/v1/bots/:id',
    api_field: 'servers',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: 'shards',
    api_get: 'https://api.botsdatabase.com/v1/bots/:id',
    api_all: null,
  },
  'botsfordiscord.com': {
    api_docs: 'https://docs.botsfordiscord.com',
    api_post: 'https://botsfordiscord.com/api/bot/:id',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://botsfordiscord.com/api/bot/:id',
    api_all: null,
  },
  'discord.boats': {
    api_docs: 'https://docs.discord.boats/',
    api_post: 'https://discord.boats/api/bot/:id',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://discord.boats/api/bot/:id',
    api_all: null,
  },
  'discord.bots.gg': {
    api_docs: 'https://discord.bots.gg/docs',
    api_post: 'https://discord.bots.gg/api/v1/bots/:id/stats',
    api_field: 'guildCount',
    api_shard_id: 'shardId',
    api_shard_count: 'shardCount',
    api_shards: null,
    api_get: 'https://discord.bots.gg/api/v1/bots/:id',
    api_all: 'https://discord.bots.gg/api/v1/bots?limit=100&sort=guildcount&order=desc',
  },
  'discordapps.dev': {
    api_docs: 'https://discordapps.dev/en-GB/posts/docs/api-v2/',
    api_post: 'https://api.discordapps.dev/api/v2/bots/:id',
    api_field: 'bot.count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://api.discordapps.dev/api/v2/bots/:id',
    api_all: 'https://api.discordapps.dev/api/v2/bots',
  },
  'discordbot.world': {
    api_docs: 'https://discordbot.world/docs',
    api_post: 'https://discordbot.world/api/bot/:id/stats',
    api_field: 'guild_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: 'shards',
    api_get: 'https://discordbot.world/api/bot/:id',
    api_all: 'https://discordbot.world/api/bots',
  },
  'discordbotdirectory.net': {
    api_docs: null,
    api_post: null,
    api_field: null,
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://discordbotdirectory.net/bots/:id',
    api_all: null,
  },
  'discordbotlist.com': {
    api_docs: 'https://docs.discordbotlist.com',
    api_post: 'https://discordbotlist.com/api/bots/:id/stats',
    api_field: 'guilds',
    api_shard_id: 'shard_id',
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://discordbotlist.com/api/bots/:id',
    api_all: 'https://discordbotlist.com/api/bots',
  },
  'discordbots.co': {
    api_docs: 'https://discordbots.co/api',
    api_post: 'https://api.discordbots.co/v1/public/bot/:id/stats',
    api_field: 'serverCount',
    api_shard_id: null,
    api_shard_count: 'shardCount',
    api_shards: null,
    api_get: 'https://api.discordbots.co/v1/public/bot/:id',
    api_all: null,
  },
  'discordextremelist.xyz': {
    api_docs: 'https://discordextremelist.xyz/docs#api-routes',
    api_post: 'https://api.discordextremelist.xyz/v2/bot/:id/stats',
    api_field: 'guildCount',
    api_shard_id: null,
    api_shard_count: 'shardCount',
    api_shards: null,
    api_get: 'https://api.discordextremelist.xyz/v2/bot/:id',
    api_all: null,
  },
  'discordlistology.com': {
    api_docs: 'https://discordlistology.com/developer/documentation',
    api_post: 'https://discordlistology.com/api/v1/bots/:id/stats',
    api_field: 'servers',
    api_shard_id: null,
    api_shard_count: 'shards',
    api_shards: null,
    api_get: 'https://discordlistology.com/api/v1/bots/:id/stats',
    api_all: null,
  },
  'disforge.com': {
    api_docs: 'https://disforge.com/developer',
    api_post: 'https://disforge.com/api/botstats/:id',
    api_field: 'servers',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: null,
    api_all: null,
  },
  'infinitybotlist.com': {
    api_docs: 'https://infinitybotlist.com/docs',
    api_post: 'https://infinitybotlist.com/api/bots/:id',
    api_field: 'servers',
    api_shard_id: null,
    api_shard_count: 'shards',
    api_shards: null,
    api_get: 'https://infinitybotlist.com/api/bots/:id/info',
    api_all: null,
  },
  'nooder.co': {
    api_docs: 'https://nooder.co/api/docs',
    api_post: 'https://nooder.co/api/bot/:id/stats',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://nooder.co/api/bot/:id',
    api_all: null,
  },
  'space-bot-list.xyz': {
    api_docs: 'https://spacebots.gitbook.io/tutorial-en/api/stats',
    api_post: 'https://space-bot-list.xyz/api/bots/:id',
    api_field: 'guilds',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://space-bot-list.xyz/api/bots/:id',
    api_all: null,
  },
  'top.gg': {
    api_docs: 'https://top.gg/api/docs',
    api_post: 'https://top.gg/api/bots/:id/stats',
    api_field: 'server_count',
    api_shard_id: 'shard_id',
    api_shard_count: 'shard_count',
    api_shards: 'shards',
    api_get: 'https://top.gg/api/bots/:id',
    api_all: null,
  },
  'voidbots.net': {
    api_docs: 'https://docs.voidbots.net/',
    api_post: 'https://voidbots.net/api/auth/stats/:id',
    api_field: 'server_count',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://voidbots.net/api/auth/info/:id',
    api_all: null,
  },
  'wonderbotlist.com': {
    api_docs: 'https://api.wonderbotlist.com/en/',
    api_post: 'https://api.wonderbotlist.com/v1/bot/:id',
    api_field: 'serveurs',
    api_shard_id: null,
    api_shard_count: 'shards',
    api_shards: null,
    api_get: 'https://api.wonderbotlist.com/v1/bot/:id',
    api_all: null,
  },
  'yabl.xyz': {
    api_docs: 'https://yabl.xyz/api',
    api_post: 'https://yabl.xyz/api/bot/:id/stats',
    api_field: 'guildCount',
    api_shard_id: null,
    api_shard_count: null,
    api_shards: null,
    api_get: 'https://yabl.xyz/api/bot/:id',
    api_all: 'https://yabl.xyz/api/bots/all',
  },
  'topcord.xyz': {
    api_docs: 'https://docs.topcord.xyz/#/',
    api_post: 'https://topcord.xyz/api/bot/stats/:id',
    api_field: 'guilds',
    api_shard_id: null,
    api_shard_count: 'shards',
    api_shards: null,
    api_get: 'https://topcord.xyz/api/bot/:id',
    api_all: null,
  }
};
```

**Review by T0TProduction — CHANGES_REQUESTED** [review]
as @MattIPv4 said, update the entire list please

File: src/fallbackListData.ts [hunk 1]
```diff
@@ -1,273 +1,304 @@
...
-    api_post: 'https://disforge.com/api/botstats/:id',
-    api_field: 'servers',
-    api_shard_id: null,
-    api_shard_count: null,
-    api_shards: null,
-    api_get: null,
-    api_all: null,
-  },
+  {
+    "arcane-center.xyz": {
```
Comment (advaith1) [inline src/fallbackListData.ts:3]: you added an additional layer of braces

**Review by advaith1 — CHANGES_REQUESTED** [review]

File: src/fallbackListData.ts (hunk 1 above)
Comment (T0TProduction) [reply src/fallbackListData.ts:3]: yeah, please keep the format that is enforced with eslint

**Review by MaximKing1 — CHANGES_REQUESTED** [review]
The API has now been changed to https://api.infinitybotlist.com, see our docs at https://docs.infinitybotlist.com for in-depth API docs 👍

**Comment by MattIPv4** [comment]
@MaximKing1 This is fallback data, if your API details have changed, you need to have them updated on BotBlock first.

**Comment by MaximKing1** [comment]
> @MaximKing1 This is fallback data, if your API details have changed, you need to have them updated on BotBlock first.

Sorry about that my bad..

**Comment by MaximKing1** [comment]
How would I do that?

**Comment by Andre601** [comment]
You go to the `#list-changes` channel on the Discord Server and mention the changes to the bot list(s) there.

**Comment by MaximKing1** [comment]
> You go to the `#list-changes` channel on the Discord Server and mention the changes to the bot list(s) there.

Thank you 👍

**Pull request closed without merging**